packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dropDownEditor.tests.js

```diff
@@ -2140,4 +2135,68 @@ QUnit.module('aria accessibility', () => {
             });
         });
     });
+
+    QUnit.module('aria-autocomplete', () => {
```

Comment:
actually "markup" tests is a more correct place for such checks

Reply:
[0ff4d1d3afcafe181316848c94f2963d6e7e6cba]